src/altinn-app-frontend/src/components/base/DatepickerComponent.test.tsx

```diff
@@ -241,4 +241,16 @@ describe('DatepickerComponent', () => {
 
     expect(handleDataChange).toHaveBeenCalledWith('');
   });
+
+  it('should have aria-describedby if textResourceBindings.description is present', () => {
+    render({ textResourceBindings: { description: 'description' }, id: 'test-id' });
+    const inputField = screen.getByRole('textbox');
+    expect(inputField).toHaveAttribute('aria-describedby', 'description-test-id');
+  });
+
+  it('should not have aria-describedby if textResources.description does not exist', () => {
+    render({ textResourceBindings: {}, id: 'test-id' });
+    const inputField = screen.getByRole('textbox');
+    expect(inputField).not.toHaveAttribute('aria-describedby', 'description-test-id');
```

Comment:
shouldn't this test be checking for `undefined-test-id` or `-test-id` ?
Or to check if the attribute exists at all despite having the given value?

Reply:
Good point. Changed test to explicitly check that attribute is not set at all (which is what we want).